catalyst-gateway/Cargo.toml

```diff
@@ -12,7 +12,7 @@ version = "0.0.1"
 authors = [
     "Steven Johnson <steven.johnson@iohk.io>"
 ]
-rust-version = "1.73"
+rust-version = "1.75"
```

Comment:
as we have decided, lets remove it 
https://github.com/input-output-hk/catalyst-ci/issues/176
```suggestion
```